### ISSUE TITLE
Make it really easy to install the examples locally

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/hvplot#820]
I've been updating the README https://github.com/holoviz/hvplot/pull/817 and thinking about what would be helpful for potential new users. I believe being able to install the example notebooks locally and spin them up in Jupyter easily would be very beneficial.

You can do it with `git clone ...` and them manually navigate. But that will clone everything and you would have to navigate a bit.

Instead I would propose some command in the terminal like

```bash
hvplot examples
```

and then a description of this process. For example under installation instructions.

This would be similar to how Panel works.

**Comment by maximlt** [comment]
Yes actually there's bits of code in the repo that should allow to do that, except that the hvplot entry point isn't configured.
